charts/delphai-deployment/templates/scaled-object.yaml

```diff
@@ -0,0 +1,25 @@
+{{ if $.Values.autoscaling.enabled }}
+{{ if index $.Values.buildMetadata.labels "com.delphai.image.release" }}
+
+{{ $queueName := tpl $.Values.autoscaling.queueName $ }}
+
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  namespace: {{ $.Release.Namespace | quote }}
+  name: {{ $.Release.Name | quote }}
+spec:
+  minReplicaCount: 1
+  maxReplicaCount: {{ $.Values.autoscaling.maxReplicaCount }}
```

Comment:
Do we need `quote` here as well?

Reply:
This field is an integer, not a string. Quoting that as a string would result in `Invalid value: "string": spec.maxReplicaCount in body must be of type integer: "string"`.

Reply:
Ah true